src/main/scala/chisel3/util/circt/IsX.scala

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.util.circt
+
+import chisel3._
+import chisel3.experimental.IntrinsicModule
+import chisel3.internal.Builder
+
+import circt.Intrinsic
+
+/** Create a module with a parameterized type which returns whether the input
+  * is a verilog 'x'.
+  */
+private class IsXIntrinsic[T <: Data](gen: T) extends IntrinsicModule("circt.isX") {
+  val i = IO(Input(gen))
+  val found = IO(Output(UInt(1.W)))
```

Comment:
```suggestion
  val in = IO(Input(gen))
  val isx = IO(Output(Bool))
```

or are these `i` and `found` tied to the intrinsic as coded in `circt.isX`?